<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QListView" name="listView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Podgląd zawartości</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Aktualne obiekty</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <column>
       <property name="text">
        <string>Nazwa</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wartość</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Lista atrybutów</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuObiekt">
    <property name="title">
     <string>Obiekt</string>
    </property>
    <addaction name="actionDodac"/>
    <addaction name="separator"/>
    <addaction name="actionUsunac"/>
   </widget>
   <widget class="QMenu" name="menuPomic">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionInformacje_o_KontynentV3D"/>
   </widget>
   <addaction name="menuObiekt"/>
   <addaction name="menuPomic"/>
  </widget>
  <action name="actionDodaj">
   <property name="text">
    <string>Usuń</string>
   </property>
  </action>
  <action name="actionUsu">
   <property name="text">
    <string>Dodaj</string>
   </property>
  </action>
  <action name="actionInformacje_o_KontynentV3D">
   <property name="text">
    <string>Informacje o KontynentV3D</string>
   </property>
  </action>
  <action name="actionDodac">
   <property name="text">
    <string>Dodaj</string>
   </property>
  </action>
  <action name="actionUsunac">
   <property name="text">
    <string>Usun</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
